lib/gui/app/components/image-selector/image-selector.jsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 'use strict'
+
+const React = require('react')
+const ReactDOM = require('react-dom')
+const propTypes = require('prop-types')
+const Color = require('color')
+
+const middleEllipsis = require('./../../utils/middle-ellipsis')
+
+const styled = require('styled-components').default
+const rendition = require('rendition')
+
+const shared = require('/./../../../../../lib/shared/units')
+const consts = require('./../../scss/main')
+const theme = require('./../../scss/modules/theme')
```

Comment:
It doesn't make sense that this is part of the `scss` directory. The `theme` module could just be in the root of the ui directory at `lib/gui/app/theme.js`.